pyvista/__init__.py

```diff
@@ -9,23 +9,272 @@
 # Load default theme.  Must be loaded first.
 from pyvista._version import __version__
 from pyvista.plotting import *
-from pyvista.utilities import *
-from pyvista.core import *
+from pyvista.utilities import (
+    # arrays,
+    cells,
+    # common,
+    # errors,
+    # features,
+    fileio,
+    # geometric_objects,
+    helpers,
+    # misc,
+    # parametric_objects,
+    reader,
```

Comment:
This is here for a `pyvista.reader.MultiBlockPlot3DReader` and a `pyvista.reader.Plot3DFunctionEnum`, both available at the top-level namespace. But perhaps downstream expects this too...